tox.ini

```
@@ -0,0 +1,21 @@
+[tox]
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+isolated_build = true
+
+
+[testenv]
+extras = test
+
+[testenv:test]
+commands =
+  pytest {posargs}
```

To ensure that you run the tests against the sdist and not the local checkout @astrofrog suggested you change this to
```suggestion
  pytest --pyargs ccdproc {toxinidir}/docs {posargs}
```